### PR TITLE
Adding the functionality to add and remove group picture

### PR DESCRIPTION
**Pull request by matheus-de-araujo**
I realized that this functionality already existed for the user profile, but did not exist for groups, so I implemented it.

Now there are two new endpoints, one for adding a photo to the group and another for removing it

An example
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "express-rate-limit": "^6.10.0",
+    "nups-whatsapp-web.js": "^1.23.1-alpha.1",
```
Comment (avoylenko) [inline package.json:16]: The project is based on official library. What's the reason you injecting your own library?

Comment (matheus-de-araujo) [reply package.json]: I'm sorry, I was doing some testing and forgot that it was in the pull request branch.
I already returned the package.json to its original form

**Comment by matheus-de-araujo** [comment]
Okay, now fix the pull request branch.

Sorry again, I was inattentive :sweat_smile:

**Comment by chrishubert** [comment]
I believe it's good to implement it - there are still too many changes in this commit, I'll start from scratch instead.
Need to add the documentation part as well if possible

**Pull request closed without merging**